core/sparqlbuilder/src/main/java/org/eclipse/rdf4j/sparqlbuilder/constraint/propertypath/builder/PropertyPaths.java

```diff
@@ -1,13 +1,13 @@
-/*
- * *****************************************************************************
- * Copyright (c) 2015 Eclipse RDF4J contributors, Aduna, and others.
+/*******************************************************************************
+ * Copyright (c) 2021 Eclipse RDF4J contributors.
```

Comment:
I think this should probably still have the old "Aduna, and others." bit. Also I think you changed the year as well.

Reply:
fixed